### DISCUSSION TITLE
How to migrate postgres 14 to 16 ?

### DISCUSSION THREAD

**Discussion by flav-code** [plausible/analytics#4261] [Self-Hosted Support] [answered]
How to migrate postgres 14 to 16 ?

**Comment by ruslandoga** [comment]
👋 @flav-code 

Here's one way: https://github.com/plausible/community-edition/blob/v2.1.1/upgrade/postgres.md

**Reply by ruslandoga** [reply, marked as answer]
Just replace 12 with 14 and 14 with 16.

**Reply by flav-code** [reply]
all good thanku
